optaplanner-core/src/test/java/org/optaplanner/core/impl/localsearch/decider/acceptor/hillclimbing/HillClimbingAcceptorTest.java

```diff
@@ -29,20 +29,20 @@
 public class HillClimbingAcceptorTest extends AbstractAcceptorTest {
 
     @Test
-    public void hillClimbingEnabled() {
+    public <Solution_> void hillClimbingEnabled() {
         HillClimbingAcceptor acceptor = new HillClimbingAcceptor();
 
-        DefaultSolverScope solverScope = new DefaultSolverScope();
+        DefaultSolverScope<Solution_> solverScope = new DefaultSolverScope<>();
```

Comment:
I'd argue that this test should use a specific generic type, so TestdataSolution:

`DefaultSolverScope<TestdataSolution> solverScope = new DefaultSolverScope<>();`

that would remove the need generifying a test method.